x-pack/plugin/core/template-resources/src/main/resources/entsearch/analytics/behavioral_analytics-events-settings.json

```diff
@@ -2,9 +2,6 @@
   "template": {
     "settings": {
       "index": {
-        "lifecycle": {
-          "name": "behavioral_analytics-events-default_policy"
-        },
```

Comment:
I'm not familiar with the behavioural analytics functionality. Apologies if this is not relevant here. 

Do the data streams created in `8.10` survive until `8.11` upon an upgrade? 
If that's the case, because the backing index template does not configure the ILM setting anymore they will be unmanaged. 

If the existing 8.10 data streams survive across Elastisearch upgrades, in order to implement a transition to DSL we'll have to continue to configure `index.lifecycle.name` and `index.lifecycle.prefer_ilm: false` (defaults to `true`) 
This way the existing `8.10` data streams will continue to receive the `index.lifecycle.name` configuration and _new_ (`8.11`) data streams will receive both ILM and `lifecycle`, together with `index.lifecycle.prefer_ilm: false` which will make them managed by DSL.

Reply:
Thank you for the explanation and catch!